src/cider/nrepl/middleware/debug.clj

```diff
@@ -305,6 +305,7 @@ this map (identified by a key), and will `dissoc` it afterwards."}
    "n" :next
    "o" :out
    "p" :inspect
+   "P" :insPect
```

Comment:
Shouldn't name the keyword differently? This seems pretty confusing to me.

Reply:
I followed suit with `:continue` and `:Continue` :)

Reply:
`:inspect-current-value` is wordy and will all indentation in `read-debug-command`

Reply:
Yeah, I just remembered this. :-) I recall I had the same misgivings about it, but space is truly at a premium.